README.md

```diff
@@ -1,11 +1,7 @@
 <!-- Oak Logo Start -->
-<h1>
```

Comment:
Hmmm... this code seems autogenerated---I am worried that this could be lost when updating---or not?